### PR TITLE
add support new device ptvo_counter_2ch

### PR DESCRIPTION
**Pull request by psike01**

### CODE REVIEW

File: src/devices/custom_devices_diy.ts [hunk 1]
```diff
@@ -1305,6 +1305,24 @@ const definitions: Definition[] = [
             await endpoint.read('genBasic', ['modelId', 'swBuildId', 'powerSource']);
         },
     },
+    {
+        zigbeeModel: ['ptvo_counter_2ch'],
+        model: 'ptvo_counter_2ch',
+        vendor: 'Custom devices (DiY)',
+        description: '[Configurable firmware](https://ptvo.info/zigbee-configurable-firmware-features/)',
```
Comment (Koenkk) [inline src/devices/custom_devices_diy.ts:1312]: Could you also create some instructions on how to create this device?

**Comment by Koenkk** [comment]
Note to self; after this merge https://github.com/Koenkk/zigbee2mqtt.io/pull/2302

File: src/devices/custom_devices_diy.ts (hunk 1 above)
Comment (psike01) [reply src/devices/custom_devices_diy.ts:1312]: > Could you also create some instructions on how to create this device?

Hi!
All can be found in this article from author :
https://modkam.ru/2021/04/29/schetchik-impulsov-zigbee/

Comment (Koenkk) [reply src/devices/custom_devices_diy.ts:1312]: It's in Russian and therefore not understandable for a large audience, do you have an English version? If not, I propose to keep this as an ext converter.

Comment (psike01) [reply src/devices/custom_devices_diy.ts:1312]: OK, I will translate it and post link

Comment (psike01) [reply src/devices/custom_devices_diy.ts:1312]: > It's in Russian and therefore not understandable for a large audience, do you have an English version? If not, I propose to keep this as an ext converter.

https://dzen.ru/a/ZUnKTY3_Dkue45gb

**Comment by Koenkk** [comment]
Looks good, will merge after https://github.com/Koenkk/zigbee2mqtt.io/pull/2302#issuecomment-1800039011 is solved.

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**